app/controllers/submissions_controller.rb

```diff
@@ -170,9 +170,16 @@ def manually_collect_and_begin_grading
                          else
                            params[:apply_late_penalty]
                          end
+    retain_existing_grading = if params[:retain_existing_grading].nil?
```

Comment:
Similar to below, here I suspect you can do a `params[:retain_existing_grading] == 'true'`. You can modify the `apply_late_penalty` setting above as well.